namespace_drift/example.yaml

```diff
@@ -1,61 +1,61 @@
 kind: Namespace
 apiVersion: v1
 metadata:
-  name: compare1
+  name: marketing-portal
   labels:
-    name: compare1
+    scope: marketing
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: nginx-deployment
-  namespace: compare1
+  name: content-delivery-app
+  namespace: marketing-portal
   labels:
-    mylabel: hello1
-    otherlabel: same-hello
+    app: web-content
+    function: delivery
 spec:
   selector:
     matchLabels:
-      app: nginx
-  replicas: 1 # tells deployment to run 1 pods matching the template
+      app: web-content
+  replicas: 1
   template:
     metadata:
       labels:
-        app: nginx
+        app: web-content
     spec:
       containers:
-      - name: nginx
+      - name: content-server
         image: nginx:1.14.2
         ports:
         - containerPort: 80
 ---
 kind: Namespace
 apiVersion: v1
 metadata:
-  name: compare2
+  name: sales-portal
   labels:
-    name: compare2
+    scope: sales
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: nginx-deployment
-  namespace: compare2
+  name: product-catalog-app
```

Comment:
This needs to match the above (content-delivery-app) so we can have a meaningful comparison.